source/containers.tex

```diff
@@ -333,7 +333,7 @@
 \begin{itemdescr}
 \pnum
 \result
-\tcode{const_iterator} if \tcode{b} is constant;
+\tcode{const_iterator} if \tcode{b} is \tcode{const};
```

Comment:
Would it be clearer to say `is of type \tcode{const X}`?

Reply:
I think that would be a bit more standardese, but I'm not sure if the full verbosity of that is needed. Perhaps `is of \tcode{const} type` would suffice.